ansible/roles/installation/docker/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Check if Docker is already installed
+  command: systemctl status docker
+  register: docker_check
+  ignore_errors: yes
+
+- name: Install Docker
+  import_tasks: install.yml
+  when: docker_check.stderr.find('service could not be found') != -1
```

Comment:
this is not language independent check. In German Ubuntu it will fail. Please at least add here TODO in case if you do not have immediate fix.

Reply:
fixed in the last commit